docs/changelog_3_4_0.rst

```diff
@@ -69,6 +69,11 @@ Modlog
 - Added option to accept a ``discord.Object`` in case creation (:issue:`4326`)
 - Added ``last_known_username`` parameter to ``create_case`` function (:issue:`4326`)
 
+Utility
+*******
+
+- Added ``redbot.core.utils.get_end_user_statement`` to attempt to fetch a cog's End User Data Statement (:issue:`4404`)
```

Comment:
a) the function is called `get_end_user_data_statement`
b) there was also `get_end_user_data_statement_or_raise` function added